ui/src/components/ResultBlock/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const ResultBlock: React.FC<{ result : any }> = ({ result }) => { 
+	return <>
+		<div className="ResultBlock">{ result.failure ? result.failure.message : "passed" }</div>
```

Comment:
This component still needs a lot of work. The intention is for it to contain the failure message, any relevant resource links, and a dropdown list of the network calls made during the lifecycle of the test.